dev-gcp-teamfamilie.yaml

```diff
@@ -48,5 +48,7 @@ spec:
   env:
     - name: APP_TEAM
       value: teamfamilie
+    - name: SPRING_PROFILES_ACTIVE
+      value: preprod
```

Comment:
Kanskje vi konsekvent skal kalle testmiljøet for dev i dette repoet, så det ikke blir så forvirrende? :)